Dockerfile

```diff
@@ -3,7 +3,7 @@
 FROM adoptopenjdk/openjdk11:jre-11.0.4_11@sha256:140ba182d696180600a2871a98f67ad0ee2d6a1e48a7c570d1c0e156860c2a9d
```

Comment:
update base image to latest version - `adoptopenjdk/openjdk11:jre-11.0.16.1_1@sha256:1e458a6a926a6265eb8abf1a4074162ee35984652b01d772f206b57406de0656`